src/uu/tail/src/tail.rs

```diff
@@ -50,6 +59,21 @@ const ABOUT: &str = "\
                      ";
 const USAGE: &str = "{} [FLAG]... [FILE]...";
 
+pub mod text {
```

Comment:
why this approach to store string this way? 
(I am not opposed, I am curious)

Reply:
Mainly for organisation so they are not floating around somewhere. Also for convenience with auto-completion, e.g. typing `text::` shows all options.

(Now that I think about it, is there already a module for common strings that can be used across uutils? If not, maybe this approach could also be used for that.)

Reply:
I don't think we have but i might be wrong.
anyway, this is an issue for another day/PR :)